gpu/command_buffer/client/implementation_base.cc

```diff
@@ -207,6 +232,14 @@ bool ImplementationBase::GetBucketContents(uint32_t bucket_id,
                           buffer.size(), buffer.shm_id(), buffer.offset());
   WaitForCmd();
   uint32_t size = *result;
+#if defined(CASTANETS)
+  // Request to synchronize shared memory of transfer buffer to get bucket data.
+  if (std::string("renderer") ==
+      base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("type")) {
```

Comment:
How about adding function for this? 
i.e.> IsRendererProcess(), IsBrowserProcess() ...

Reply:
It's a great idea. I'll try to make another patch for it because this function has been applied to other patches before.

Reply:
OK :)

Reply:
Just a clarification, RequestSyncTransferBuffer() is needed both here and WaitForCmd() as well ?

Reply:
> i.e.> IsRendererProcess(), IsBrowserProcess() ...

It can be a good way for code cleanup. But I think it is necessary to make it work properly without checking the process type anywhere.

Reply:
@suyambulingamrm Sorry, I missed your question. Sync of WaitForCmd is for the result of commands and here is for bucket data. So both of them are needed.
But this request sync message was removed and changed on latest PR.
https://github.com/Samsung/Castanets/pull/162

Reply:
Thanks for the explanation @Sunwoo-Nam.. I got to know later :)